packages/visualizations/src/components/Table/Cell/Format/GeoFormat.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+    import Map from '../../../utils/Map';
+    import type { MapOptions, MapData } from '../../../utils/Map/types';
```

Comment:
I think we can setup `src` or even `utils` aliases same as we do in `visualizations-react`

Reply:
Done https://github.com/opendatasoft/ods-dataviz-sdk/pull/245/commits/f24f939ad6a953924473c5e72d9182cde9428593